ansible/roles_ocp_workloads/ocp4_workload_amq_streams_dev_exp/README.adoc

```diff
@@ -0,0 +1,47 @@
+= ocp4-workload-amq-streams-dev-experienced
+
+Role for GPTE AMQ Streams Development Experienced course
+
+== Role overview
+
+=== Deploy a Workload with the `ocp-workload` playbook [Mostly for testing]
+
+----
+TARGET_HOST="bastion.dev4.openshift.opentlc.com"
+SSH_USER="opentlc-mgr"
+SSH_PRIVATE_KEY="id_rsa"
+OCP_USERNAME="johndoe-redhat.com"
+WORKLOAD="ocp4_workload_amq_streams_dev_exp"
+GUID="testguid"
+
+# a TARGET_HOST is specified in the command line, without using an inventory file
+ansible-playbook -i ${TARGET_HOST}, ./configs/ocp-workloads/ocp-workload.yml \
+    -e"ansible_user=${SSH_USER}" \
+    -e"ansible_ssh_private_key_file=~/.ssh/${SSH_PRIVATE_KEY}" \
+    -e"ocp_username=${OCP_USERNAME}" \
+    -e"ocp_workload=${WORKLOAD}" \
+    -e"guid=${GUID}" \
+    -e"ocp_user_needs_quota=true" \
+    -e"ACTION=create"
```

Comment:
This way of testing has been around and copied / pasted in basically every workload.

IMO It feels a bit hacky and not in the vein of how we generally use config in agnosticd.

We now have a config, ocp-workloads that does that and you don't have to touch the inventory. So it goes like this:

```yaml
cloud_provider: none
env_type: ocp-workloads

ocp_workloads:
  - ocp4_workload_pipelines

target_host: bastion.dev.mycluster.mydomain.com

#become_override: false

# If the ocp-workload supports it, you should specify the OCP user:
# ocp_username: myuser

# Usually the ocp-workloads want a GUID also:
# guid: changeme
```

`target_host` can be a either a string or a dictionary:

```yaml
target_host:
  ansible_host: bastion.dev.mycluster.mydomain.com
  ansible_port: 22
  ansible_user: ec2-user
  #ansible_ssh_private_key_content: "{{ ssh_private_key_content }}"
  ansible_ssh_private_key_file: ~/.ssh/admin_key.pem
  #ansible_ssh_extra_args:  ...
```

And then  `ansible-playbook main.yml -e @vars.yml`